runtime/databricks/automl_runtime/forecast/prophet/model.py

```diff
@@ -117,35 +136,45 @@ def predict_timeseries(self, horizon: int = None) -> pd.DataFrame:
         """
```

Comment:
I felt the name `predict_timeseries` is not clear enough. It's not obvious how it's different than the `predict` method. Maybe we can call it `forecast` and clarify in the docstring that this method makes multiple-step forecasting and `horizon` is the number of time steps.

Reply:
The original design for `predict_timeseries` is basically the same with `predict` but only take the `horizon`. Right now the major difference is that `predict_timeseries` keeps the extra columns from prophet. We can keep this now for backward compatibility.

Reply:
i just realized that `predict_timeseries` provides predictions for the given horizon as well as in-sample data. Shall we add another method `forecast` that only does out-of-sample forecasting?

Reply:
Let's do this as a follow up.